index.bs

```diff
@@ -2480,10 +2480,7 @@ To <dfn data-lt='Fire a "zoomchange" event'>fire a {{ZoomChangeEvent}}</dfn> |e|
 
 [Exposed=Window, IsolatedContext]
 interface ContentLoadEvent : Event {
-  constructor(DOMString type, optional ContentLoadEventInit eventInitDict = {});
-};
-
-dictionary ContentLoadEventInit: EventInit {
+  constructor(DOMString type);
```

Comment:
This should pass the generic `EventInit` options parameter through to the main `Event` constructor.